CSS/css_completions.py

```diff
@@ -426,7 +426,7 @@ class CSSCompletions(sublime_plugin.EventListener):
     regex = None
 
     def on_query_completions(self, view, prefix, locations):
-        selector_scope = "source.css - meta.selector.css"
+        selector_scope = "source.css - meta.selector.css, text.html meta.attribute-with-value.style.html string.quoted punctuation.definition.string.end.html"
```

Comment:
that would work too, and is probably clearer (y)

my solution worked because the rest of the attribute is already covered by the `source.css`[...] selector.